Web Development/JS-Game-Project-Frogger/README.md

```diff
@@ -0,0 +1,11 @@
+# Frogger Game
+## Description
+This is Frogger Game
+A retro grid-based game in JavaScript, HTML and CSS
+
```

Comment:
if any installation and Dependencies are required then mention here by making new row